conanfile.py

```diff
@@ -150,9 +150,14 @@ def _generate_about_versions(self, location):
             conan_installs.append([dependency.ref.name,dependency.ref.version])
 
         #list of python installs
-        import pkg_resources
-        for package in pkg_resources.working_set:
-            python_installs.append([package.key, package.version])
+        python_ins_cmd = f""" python -c "import pkg_resources; import os; [open(os.path.join("'"{self.recipe_folder}"'", "'"info.txt"'"), "'"a"'").write(s.key+"'","'"+ s.version+"'";"'") for s in pkg_resources.working_set]" """
+        self.run(python_ins_cmd, run_environment= True, env = "conanrun",  output=True)
+        with open(os.path.join(self.recipe_folder, "info.txt"), "r") as f:
```

Comment:
A bit of defensive programming, first check for existence before opening. I'm okay if this fails gracefully and no packages are listed when in development mode `self.in_local_cache` is `False` when running in development mode